.rubocop.yml

```diff
@@ -14,6 +14,11 @@ Metrics/AbcSize:
 Metrics/ClassLength:
   Max: 1300
 
+# Offense count: 1
```

Comment:
what's the purpose of this configuration change on rubocop?

Reply:
The module in exception_detector.rb became too large...